gulpfile.js

```diff
@@ -85,6 +85,7 @@ declareExtension('amp-list', '0.1', false, 'NO_TYPE_CHECK');
 declareExtension('amp-live-list', '0.1', true);
 declareExtension('amp-mustache', '0.1', false, 'NO_TYPE_CHECK');
 declareExtension('amp-o2-player', '0.1', false, 'NO_TYPE_CHECK');
+declareExtension('amp-ooyala-player', '0.1', false, 'NO_TYPE_CHECK');
```

Comment:
please enable type check by removing 'NO_TYPE_CHECK' and running `gulp check-type` to ensure all is good.